charts/cluster-addons/templates/intel-device-plugin.yaml

```diff
@@ -0,0 +1,84 @@
+{{- if .Values.intelDevicePlugin.enabled }}
+---
+apiVersion: source.toolkit.fluxcd.io/v1
+kind: GitRepository
+metadata:
+  name: intel-device-plugin-kustomization-repo
+  namespace: default
+spec:
+  interval: 5m
+  url: {{ .Values.intelDevicePlugin.repo.url }}
+  ref:
+    tag: {{ .Values.intelDevicePlugin.repo.tag }}
+---
+apiVersion: kustomize.toolkit.fluxcd.io/v1
+kind: Kustomization
+metadata:
+  name: intel-device-plugin-kustomization
+  namespace: default
+spec:
+  interval: 10m
+  sourceRef:
+    kind: GitRepository
+    name: intel-device-plugin-kustomization-repo
+  prune: true
+  timeout: 1m
+  kubeConfig:
+    secretRef:
+      name: {{ .Values.clusterName }}-kubeconfig
+  path: ./gpu-plugin
+  patches:
+    - patch: |-
+        apiVersion: deviceplugin.intel.com/v1
+        kind: GpuDevicePlugin
+        metadata:
+          name: gpudeviceplugin-config
+        spec:
+          image: {{ .Values.intelDevicePlugin.gpuPlugin.image }}
+          sharedDevNum: {{ .Values.intelDevicePlugin.gpuPlugin.sharedDevNum }}
+          preferredAllocationPolicy: {{ .Values.intelDevicePlugin.gpuPlugin.preferredAllocationPolicy }}
+          logLevel: {{ .Values.intelDevicePlugin.gpuPlugin.logLevel }}
```

Comment:
Lets add tolerations in here:
https://github.com/intel/intel-device-plugins-for-kubernetes/blob/199dfba978f1190b04d97028e0d7c99fd66bbb86/pkg/apis/deviceplugin/v1/gpudeviceplugin_types.go#L44